### PR TITLE
Web snackbar

### LINKED ISSUE #5: Show bookmarking/subscribing not available on web
The bookmark button or subscribe button do not work on web, because they store information on sqlite.

We should handle these buttons on web and instead show a snackbar or a popup saying that this feature is not available on web.

### PR DESCRIPTION
**Pull request by AlKun25**
Resolved #5
Snackbar added
Attaching pics:
image
image

### CODE REVIEW

File: lib/ui/tiles/subscribe_button.dart [hunk 1]
```diff
@@ -12,7 +13,8 @@ class SubscribeButton extends StatefulWidget {
 
 class _SubscribeButtonState extends State<SubscribeButton> {
   bool marked;
-
+  final _snackBar =
+      SnackBar(content: Text('Web does not support this feature'));
```
Comment (mannprerak2) [inline lib/ui/tiles/subscribe_button.dart]: Update message: Subscribing is not supported on web.
Also add a "Bell" icon along with the message.

File: lib/ui/tiles/bookmark_button.dart [hunk 2]
```diff
@@ -12,7 +13,8 @@ class BookmarkButton extends StatefulWidget {
 
 class _BookmarkButtonState extends State<BookmarkButton> {
   bool marked;
-
+  final _snackBar =
+      SnackBar(content: Text('Web does not support this feature'));
```
Comment (mannprerak2) [inline lib/ui/tiles/bookmark_button.dart]: Update message: Bookmarking is not supported on web.
Also add a "Bookmark" icon along with the meessage.

**Review by mannprerak2 — CHANGES_REQUESTED** [review]

File: lib/ui/tiles/subscribe_button.dart (hunk 1 above)
Comment (AlKun25) [reply lib/ui/tiles/subscribe_button.dart]: Done

File: lib/ui/tiles/bookmark_button.dart (hunk 2 above)
Comment (AlKun25) [reply lib/ui/tiles/bookmark_button.dart]: Done

**Review by mannprerak2 — APPROVED** [review]

**Comment by mannprerak2** [comment]
Will merge this after your other PR.

**Comment by AlKun25** [comment]
Ok

**Comment by mannprerak2** [comment]
Please pull changes from upstream/master.
(Set this repository as upstream)
and fix the errors (if any)

**Review by AlKun25 — COMMENTED** [review]
Updated the branch with the upstream

**Comment by AlKun25** [comment]
The link for the website doesn't seem to be updated, with the most recent changes

**Comment by mannprerak2** [comment]
@AlKun25 
It's working now check it out.

**Comment by AlKun25** [comment]
@mannprerak2
It would be great if you could label my merged PRs as "hacktoberfest-accepted". I guess they have made a new condition for considering PRs.

**Comment by mannprerak2** [comment]
Hi @AlKun25, I don't think this is required since this repository has a hacktoberfest label and this PR has been merged.
This should be be counted automatically according to the rules.
Please check if this PR is visible in your Dashboard as "Pending", If it is, then after the review period it wiill get accepted.

**Comment by AlKun25** [comment]
Thanks for the clarification!

**Pull request merged**